pkg/router/template/router.go

```diff
@@ -171,6 +176,17 @@ func newTemplateRouter(cfg templateRouterCfg) (*templateRouter, error) {
 	return router, nil
 }
 
+func matchString(pattern, s string) bool {
```

Comment:
@smarterclayton 
I have added three more utility functions to help with any faster evaluations.. regex helper should still be there for advanced use cases I believe.